_projects/quickstart_green_mode.md

```diff
@@ -0,0 +1,96 @@
+---
+collaborating_projects:
+  - coala-quickstart
+  - coala
+desc: "Generate project compatible config files."
+difficulty: medium
+initiatives:
+- GSoC
+- RGSoC
+issues:
+markdown: quickstart_green_mode.md
+mentors:
+  - satwikkansal
+  - jayvdb
+name: "Quickstart Green Mode"
+requirements:
+- "The applicant should have at least one patch accepted to coala-quickstart
+  repository."
+- "The participant should have an understanding of coala's structure and
+  config files."
+- "The proposal needs to contain a clear concept."
+tags:
+- coala-quickstart
+- coala-ci
+---
+
+coala doesn't help the user unless the user is familiar with its settings and
+bears. coala-quickstart is the project to handle this situation by asking the
+users general questions about their project and producing a config file to lint
+their code.
+
+Currently coala-quickstart asks the user a lot of questions, and usually
+generates a `.coafile` which causes coala-ci to generate errors.
+There is a need for production of config files which yield 'green' results
+which adapt the bears and their settings according to the project. This mode
+would run coala many times in order to answer questions itself, altering bear
+settings each time to find the set of settings which produces the least amount
+of errors, and where necessary adds ignore for the remaining files.
+
+This will lead to massive adoption of coala across organizations and user
+projects as this would initially configure the bears to agree with the repo
```

Comment:
Sentence too long, maybe break it into two? 

Also, the word "massive" is bit overqualified here. Maybe use "easier adoption" or "increase is adoption"